src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
The Main class should not begin with an empty line according to the checklist. Please remove the empty line at the beginning of the class definition.